.binder/postBuild

```diff
@@ -0,0 +1,33 @@
+MONGOTOOLS=mongodb-database-tools-ubuntu1804-x86_64-100.7.4
+MONGOTOOLS_FILE=$MONGOTOOLS.tgz
+MONGOTOOLS_LINK=https://fastdl.mongodb.org/tools/db/$MONGOTOOLS_FILE
+
+MONGODB=mongodb-linux-x86_64-ubuntu1804-6.0.8
+MONGODB_FILE=$MONGODB.tgz
+MONGO_LINK=https://fastdl.mongodb.org/linux/$MONGODB_FILE
+
+# Download the MongoDB and Mongorestore
+wget $MONGO_LINK 
+tar -zxvf $MONGODB_FILE
+
+wget $MONGOTOOLS_LINK
+tar -zxvf $MONGOTOOLS_FILE
+
+# Download mongodb data
+wget https://superduperdb-public.s3.eu-west-1.amazonaws.com/collab_data.zip
+unzip collab_data.zip -d dump
+rm -f collab_data.zip
+
+# Move mongodb tools into a seperate directory
+mkdir -p $HOME/.tools
+mv $MONGODB $HOME/.tools/mongodb
+mv $MONGOTOOLS $HOME/.tools/mongotools
+
+# Create a startup script which will be triggering mongodb, 
+# Whenever a notebook is launched.
+mkdir -p $HOME/.ipython/profile_default/startup
+cp .binder/start.py $HOME/.ipython/profile_default/startup
+cp .binder/start.sh $HOME/.ipython/profile_default/startup
```

Comment:
```suggestion
```
Duplicate.

Reply:
we need start.sh 
please check start.py